src/containers/booksList.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Book from '../components/book';
+
```

Comment:
Let's move our `mapStateToProp` in `index.js` down here.
![image](https://user-images.githubusercontent.com/31897434/79174924-22f20500-7df4-11ea-94e0-674018fa8933.png)